source/lambda/apiprocessor/lambda_function.py

```diff
@@ -21,12 +22,30 @@ def lambda_handler(event, context):
         request["outputTable"] = os.environ['OUTPUT_TABLE']
         request["documentsTable"] = os.environ['DOCUMENTS_TABLE']
 
+        # search Elasticsearch handler
         if(event['resource'] == '/search'):
             if('queryStringParameters' in event and 'k' in event['queryStringParameters']):
                 request["keyword"] = event['queryStringParameters']['k']
                 if('documentId' in event['queryStringParameters']):
                     request["documentId"] = event['queryStringParameters']['documentId']
                 result = search(request)
+    
+        # search Kendra if available
+        elif(event['resource'] == '/searchkendra' and event['httpMethod'] == 'POST'):
+            if 'KENDRA_INDEX_ID' in os.environ :
+                kendraClient = KendraHelper()
+                result = kendraClient.search(os.environ['KENDRA_REGION'],
```

Comment:
Do we need to pass the region from Kendra env variable? 
Lambda itself sets several environment variables during initialization (including region)
You can view the documentation here: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime